aliases/available/vim.aliases.bash

```diff
@@ -5,15 +5,19 @@ VIM=$(command -v vim)
 GVIM=$(command -v gvim)
 MVIM=$(command -v mvim)
 
-[[ -n $VIM ]] && alias v=$VIM
+# open the vim help in fullscreen incorporated from
+# https://stackoverflow.com/a/4687513
+alias vimh='${VIM} -c ":h | only"'
```

Comment:
I think you should put this inside a `if [[ -n $VIM]]` like what's being done with the `v` alias

Reply:
Oh, fair comment. Will do and hopefully as you Invision it